docs/docs/ref-06-dom-differences.md

```diff
@@ -10,6 +10,7 @@ React has implemented a browser-independent events and DOM system for performanc
 
 * All DOM properties and attributes (including event handlers) should be camelCased to be consistent with standard JavaScript style. We intentionally break with the spec here since the spec is inconsistent. **However**, `data-*` and `aria-*` attributes [conform to the specs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#data-*) and should be lower-cased only.
 * The `style` attribute accepts a JavaScript object with camelCased properties rather than a CSS string. This is consistent with the DOM `style` JavaScript property, is more efficient, and prevents XSS security holes.
+* Since JSX is JavaScript, identifiers such as `class` and `for` are discouraged as XML attribute names. Instead, JSX (React DOM) components expect DOM property names like `className` and `htmlFor`, respectively.
```

Comment:
Should note that known DOM tags use `className` and `htmlFor` but custom elements (aka. web components) must use `class` and `for` directly.  Example: `<my-tag class="foo" />`